### PR TITLE
cifar10

### PR DESCRIPTION
**Pull request by dczifra**

### CODE REVIEW

File: augmentation.py [hunk 1]
```diff
@@ -42,10 +42,33 @@ def get_augmentations(args):
...
+class TestAugmentation(object):
+    def __init__(self, args):
+        if(args.dataset=="imagenet"):
+            mean = [0.485, 0.456, 0.406]
+            std = [0.229, 0.224, 0.225]
+        elif(args.dataset=="cifar10"):
+            mean= [0.4914, 0.4822, 0.4465]
+            std = [0.2023, 0.1994, 0.2010]
+
+        normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225])
```
Comment (gergopool) [inline augmentation.py]: itt nem mean es std?

File: datasets.py [hunk 2]
```diff
@@ -32,10 +34,94 @@ def __getitem__(self, index):
...
+
+def get_train_with_val_augmentation(args, num_tasks, global_rank):
+    val_aug = TestAugmentation(args).aug
+
+    if args.dataset == "imagenet_lmdb":
+        dataset_val = ImageNetLMDB(args.data_path, 'val.lmdb', val_aug)
+    elif args.dataset == "imagenet":
+        dataset_val = ImageNet(args.data_path, val_aug, train=True)
+    elif args.dataset == "cifar10":
+        dataset_val = CIFAR10(args.data_path, val_aug, train=True)
```
Comment (gergopool) [inline datasets.py]: itt train=True? Miert dataset_val a valtozo neve?

File: widen_resnet.py [hunk 3]
```diff
@@ -178,7 +179,7 @@ def __init__(
         self.base_width = width_per_group
 
         num_out_filters = width_per_group * widen
-        self.conv1 = nn.Conv2d(3, num_out_filters, kernel_size=7, stride=2, padding=3,
+        self.conv1 = nn.Conv2d(3, num_out_filters, kernel_size=conv1_size, stride=2, padding=3,
```
Comment (gergopool) [inline widen_resnet.py]: Ez keves valtoztatas cifarhoz, ha kernel_size==3, attol meg nem lesz cifaros resnet18 halo. Itt stride=1 kell, a self.maxpool pedig nem kell.

Comment (gergopool) [reply widen_resnet.py]: Azt hiszem ennyi, de biztos ami biztos, vessuk ossze egy masik repoval, nehogy azon menjen el, hogy rosszul tudom.

File: augmentation.py (hunk 1 above)
Comment (dczifra) [reply augmentation.py]: Köszi, igen ez bug...

File: datasets.py (hunk 2 above)
Comment (dczifra) [reply datasets.py]: Igazad van, itt a megnevezés félrevezető, dataset_train-re átneveztem.

File: widen_resnet.py (hunk 3 above)
Comment (dczifra) [reply widen_resnet.py]: Köszi, most betettem az is_cifar10 változót, az első convolúciós layer kernel, stride, padding paramétere lett más, plusz kivettem a max_poolt.
